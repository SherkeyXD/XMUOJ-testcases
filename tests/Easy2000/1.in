nbb
oyp
ogr
vqf
ads
fzh
ygu
xqy
hkl
nbu
tjo
aif
sse
lyo
ssy
zbd
aar
lea
mic
bhf
hrp
jez
xgr
qrf
xth
fuf
yik
een
fyn
jwp
mux
syv
wzj
gdp
aqv
asi
fir
xkh
qla
pdq
voh
xzt
jcp
vgg
sbi
neo
ppj
udy
vpk
gmh
aor
xyr
vix
hat
utm
rpp
bco
cwa
hge
yxt
ebe
jnu
paw
keq
cht
tyi
jwf
hvy
phr
fwd
tce
kkd
lxg
ije
mkv
iba
sdw
knr
nkh
tvw
meq
kkk
aqq
qvp