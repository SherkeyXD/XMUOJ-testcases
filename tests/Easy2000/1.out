b b n
o p y
g o r
f q v
a d s
f h z
g u y
q x y
h k l
b n u
j o t
a f i
e s s
l o y
s s y
b d z
a a r
a e l
c i m
b f h
h p r
e j z
g r x
f q r
h t x
f f u
i k y
e e n
f n y
j p w
m u x
s v y
j w z
d g p
a q v
a i s
f i r
h k x
a l q
d p q
h o v
t x z
c j p
g g v
b i s
e n o
j p p
d u y
k p v
g h m
a o r
r x y
i v x
a h t
m t u
p p r
b c o
a c w
e g h
t x y
b e e
j n u
a p w
e k q
c h t
i t y
f j w
h v y
h p r
d f w
c e t
d k k
g l x
e i j
k m v
a b i
d s w
k n r
h k n
t v w
e m q
k k k
a q q
p q v
